frontend/src/metabase/browse/components/BrowseMetrics.tsx

```diff
@@ -0,0 +1,84 @@
+import { t } from "ttag";
+
```

Comment:
Wrong cursor shown (should be `pointer`)

https://github.com/user-attachments/assets/060100b1-19fe-40d0-836b-572118319925